_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+|`sql.insights.anomaly_detection.latency_threshold`                      | `50 ms`       | The latency threshold that triggers monitoring a statement fingerprint for unusually slow execution.                                                                                          | Statement executions                 |
+|`sql.insights.anomaly_detection.memory_limit`                           | `1` MiB       | The maximum amount of memory allowed for tracking statement latencies.                                                                                                                        | Statement executions                 |
+|`sql.insights.latency_threshold`                                        | `100 ms`      | The threshold at which the contention duration of a contended transaction is considered **High Contention** or statement execution is flagged for insights.                                   | Statement and Transaction executions |
+|`sql.insights.high_retry_count.threshold`                               | `10`          | The threshold at which a retry count is considered **High Retry Count**.                                                                                                                      | Statement executions                 |
+|`sql.insights.execution_insights_capacity`                              | `1000`        | The maximum number of execution insights stored in each node.                                                                                                                                 | Statement executions                 |
+|`sql.contention.event_store.capacity`                                   | `64 MiB`      | The in-memory storage capacity of the contention event store in each nodes.                                                                                                                   | Transaction executions               |
+|`sql.contention.event_store.duration_threshold`                         | `0`           | The minimum contention duration to cause contention events to be collected into the `crdb_internal.transaction_contention_events` table.                                                      | Transaction executions               |
+
+#### Detect slow executions
+
+There are two different methods for detecting slow executions. By default, they are both enabled and you can configure them based on your workload.
+
+The first method flags all executions running longer than `sql.insights.latency_threshold`. This is analogous to checking the [slow query log]({{ link_prefix }}logging-use-cases.html#sql_perf).
```

Comment:
The cluster settings mentioned in this section should each link to the anchor for that setting on the 'Cluster Settings' page.  The reason for doing this is that folks could get sent an anchor link directly to this 'Detect slow executions' section of the page, and they should be able to click directly from this section to see the docs for those settings, without doing additional searching on the page

Reply:
I noticed a cluster setting is linked below, so I'd modify the above to say "the first mention of each setting" should have a link